app/src/test/java/com/example/concordia_campus_guide/ModelTests/TestUtils.java

```diff
@@ -118,5 +121,45 @@ public Shuttle getShuttle2(){
         return  shuttle2;
     }
 
+    public static DirectionWrapper getDirectionWrapper1(){
+        DirectionWrapper directionWrapper1 = new DirectionWrapper();
+        Direction direction1 = new Direction();
+        direction1.setDuration(60.0);
+        direction1.setDescription("Go Left");
+
+        directionWrapper1.setDirection(direction1);
+        return directionWrapper1;
+    }
+
+    public static DirectionWrapper getDirectionWrapper2(){
+        DirectionWrapper directionWrapper2 = new DirectionWrapper();
+        Direction direction1 = new Direction();
+        direction1.setDuration(120.0);
+        direction1.setDescription("Go Right");
+
+        directionWrapper2.setDirection(direction1);
+        return directionWrapper2;
+    }
+
+    public static List<DirectionWrapper> getListOfDirectionWrappers(){
+        return new ArrayList<>(Arrays.asList(getDirectionWrapper1(), getDirectionWrapper2()));
+    }
+
+    public static com.google.android.gms.maps.model.LatLng getLatLng1(){
```

Comment:
I see no reason for this to be in a separate Utils class, in general I would discourage the use of utils methods because they become poluted and can easily become a code smell

Reply:
Okay, I see what you're trying to get at!